README.md

```diff
@@ -384,3 +386,48 @@ const assert = v.assert(v.range([-10, 10]));
 assert(4); // pass
 assert(-100); // fail
 ```
+
+
+## Custom validators
+One of the primary goals of Fusspot is to be customizable out of the box. There are multiple ways to which one can create a custom validator. 
+
+### Simple
+A simple custom validator is nothing but a function which accepts the input `value` and returns a `string` if the input `value` doesn't pass the test. The `string` returned would then be used to show a helpful error to users. Below is an example of a path validator for node environment.
```

Comment:
Let's make it clear that the returned `string` should be a noun phrase describing the expected value type, and will be inserted into the sentence `value must be a(n) {description}`.